examples/pending-order-reminder/Send Reminder on Approval Pending Orders.md

```diff
@@ -0,0 +1,35 @@
+# Send reminder on approval pending sales orders
+
+This example illustrates the use of the `sap:Client` in Ballerina to interact with S/4HANA APIs. Specifically, it demonstrates how to send a reminder email for sales orders that are pending approval.
+
+## Prerequisites
+
+### 1. Setup S/4HANA API
```

Comment:
```suggestion
### 1. Setup the S/4HANA API
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-sap/pull/10